app/models/list.rb

```diff
@@ -1,3 +1,5 @@
 class List < ApplicationRecord
   belongs_to :user
+  has_many :list_ingredients
+  has_many :ingredients, through: :list_ingredients
 end
```

Comment:
You'll need to change the relationship in "ingredient" model to accomadate for "list_ingredients"

Reply:
done! thanks for reminding me!